test/unit/specs/addressbook.spec.js

```diff
@@ -48,7 +48,7 @@ describe("Addressbook", () => {
       persistent_peers: ["123.456.123.456"]
     })
     let node = await addressbook.pickNode()
-    expect(node).toBe("123.456.123.456:46657")
+    expect(node).toMatchSnapshot()
```

Comment:
curious as to the benefit of this way over the previous. previous is more explicit no? whereas matching snapshot is more easily overwritten with the -u tag meant for another test. are there other benefits?

Reply:
While refactoring I noticed the benefit of updating snapshot with `-u`. A downside could be that one could miss a  `undefined` snapshot